### PR TITLE
WIP for android white screen error

### PR DESCRIPTION
**Pull request by bionara**
# Description

I suspect our errors are caused by either transitions clashing with http (see comment) and/or related to WebView.

[comment]: <> (Description of the feature or fix)
[comment]: <> (Include link to Figma designs, if relevant)
[comment]: <> (Include link to Notion ticket, if relevant)

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

**Review by julienlavigne — APPROVED** [review]
Looks legit on the basis of the slack comment but I didn't dig into the linked github issue

**Review by selvach — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,8 +54,8 @@ references:
   # Branch filter
   master_filter: &master_filter
     branches:
-      only: master
-
```
Comment (selvach) [inline .circleci/config.yml:58]: Unnecessary, but fine :)

File: android/gradle.properties [hunk 2]
```diff
@@ -28,7 +28,9 @@ android.enableJetifier=true
 
 # Disable daemon in CircleCI
 # https://support.circleci.com/hc/en-us/articles/360021812453-Common-Android-memory-issues
+# and https://stackoverflow.com/questions/59044161/expiring-daemon-because-jvm-heap-space-is-exhausted
 org.gradle.daemon=false
+org.gradle.jvmargs=-Xmx4096m
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.33.1
+FLIPPER_VERSION=0.93.0
```
Comment (selvach) [inline android/gradle.properties:36]: Should not affect OTA, I believe.

File: src/components/WebView.tsx [hunk 3]
```diff
@@ -9,7 +9,18 @@ export class WebView extends React.Component<IProps, object> {
...
+    // https://github.com/react-native-webview/react-native-webview/issues/1069
+    return (
+      <NativeWebView
+        {...this.props}
+        androidLayerType="software"
+        automaticallyAdjustContentInsets={false}
+        onMessage={this.onMessage}
+        ref={this.refWebView}
+        style={{ minHeight: 1, opacity: 0.99 }}
+      />
```
Comment (selvach) [inline src/components/WebView.tsx:22]: Are you able to test this on an iOS ?

File: .circleci/config.yml (hunk 1 above)
Comment (bionara) [reply .circleci/config.yml:58]: Fallout from debug. At least it’s consistent now lol

**Pull request merged**